components/brave_new_tab_ui/reducers/new_tab_reducer.ts

```diff
@@ -179,6 +180,10 @@ export const newTabReducer: Reducer<NewTab.State | undefined> = (state: NewTab.S
       }
       break
 
+    case types.BUY_BITCOIN_DOT_COM_CRYPTO:
```

Comment:
I need to move this and its respective action and type constant to their own files, like gemini and binance

Reply:
moved